compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/Keyword.java

```diff
@@ -0,0 +1,141 @@
+    ISOLATED(LexerTerminals.ISOLATED),
+
+    RETURNS(LexerTerminals.RETURNS),
+    RETURN(LexerTerminals.RETURN),
+    EXTERNAL(LexerTerminals.EXTERNAL),
+    TRUE(LexerTerminals.TRUE),
+    FALSE(LexerTerminals.FALSE),
+    IF(LexerTerminals.IF),
+    ELSE(LexerTerminals.ELSE),
+    WHILE(LexerTerminals.WHILE),
+    CHECK(LexerTerminals.CHECK),
+    CHECKPANIC(LexerTerminals.CHECKPANIC),
+    PANIC(LexerTerminals.PANIC),
+    CONTINUE(LexerTerminals.CONTINUE),
+    BREAK(LexerTerminals.BREAK),
+    TYPEOF(LexerTerminals.TYPEOF),
+    IS(LexerTerminals.IS),
+    NULL(LexerTerminals.NULL),
+    LOCK(LexerTerminals.LOCK),
+    FORK(LexerTerminals.FORK),
+    TRAP(LexerTerminals.TRAP),
+    IN(LexerTerminals.IN),
+    FOREACH(LexerTerminals.FOREACH),
+    TABLE(LexerTerminals.TABLE),
+    KEY(LexerTerminals.KEY),
+    LET(LexerTerminals.LET),
+    NEW(LexerTerminals.NEW),
+    FROM(LexerTerminals.FROM),
+    WHERE(LexerTerminals.WHERE),
+    SELECT(LexerTerminals.SELECT),
+    START(LexerTerminals.START),
+    FLUSH(LexerTerminals.FLUSH),
+    CONFIGURABLE(LexerTerminals.CONFIGURABLE),
+    WAIT(LexerTerminals.WAIT),
+    DO(LexerTerminals.DO),
+    TRANSACTION(LexerTerminals.TRANSACTION),
+    TRANSACTIONAL(LexerTerminals.TRANSACTIONAL),
+    COMMIT(LexerTerminals.COMMIT),
+    ROLLBACK(LexerTerminals.ROLLBACK),
+    RETRY(LexerTerminals.RETRY),
+    ENUM(LexerTerminals.ENUM),
+    BASE16(LexerTerminals.BASE16),
+    BASE64(LexerTerminals.BASE64),
+    MATCH(LexerTerminals.MATCH),
+    CONFLICT(LexerTerminals.CONFLICT),
+    LIMIT(LexerTerminals.LIMIT),
+    JOIN(LexerTerminals.JOIN),
+    OUTER(LexerTerminals.OUTER),
+    EQUALS(LexerTerminals.EQUALS),
+    CLASS(LexerTerminals.CLASS),
+    ORDER(LexerTerminals.ORDER),
+    BY(LexerTerminals.BY),
+    ASCENDING(LexerTerminals.ASCENDING),
+    DESCENDING(LexerTerminals.DESCENDING),
+
+    // Type keywords
+    INT(LexerTerminals.INT),
+    BYTE(LexerTerminals.BYTE),
+    FLOAT(LexerTerminals.FLOAT),
+    DECIMAL(LexerTerminals.DECIMAL),
+    STRING(LexerTerminals.STRING),
+    BOOLEAN(LexerTerminals.BOOLEAN),
+    XML(LexerTerminals.XML),
+    JSON(LexerTerminals.JSON),
+    HANDLE(LexerTerminals.HANDLE),
+    ANY(LexerTerminals.ANY),
+    ANYDATA(LexerTerminals.ANYDATA),
+    NEVER(LexerTerminals.NEVER),
+    VAR(LexerTerminals.VAR),
+    MAP(LexerTerminals.MAP),
+    FUTURE(LexerTerminals.FUTURE),
+    TYPEDESC(LexerTerminals.TYPEDESC),
+    ERROR(LexerTerminals.ERROR),
+    STREAM(LexerTerminals.STREAM),
+    READONLY(LexerTerminals.READONLY),
+    DISTINCT(LexerTerminals.DISTINCT),
+    FAIL(LexerTerminals.FAIL),
+    MODULE(LexerTerminals.MODULE),
+    VARIABLE(LexerTerminals.VARIABLE),
+    DEFAULT("");
+
+    private String keyword;
+
+    Keyword(String keyword) {
+        this.keyword = keyword;
+    }
+
+    public String toString() {
+        return keyword;
+    }
+
+    public static Keyword getValue(String value) {
+        return keywordsMap.getOrDefault(value, DEFAULT);
```

Comment:
Here shouldn't the default be the actual en key?
Can you test this with missing mapping property file

Reply:
if the value (like `remove`) is not in the `keywordMap` instead of getting error it will return the `DEFAULT`. Returning default is handled here -> 84391b03d60675b3c2744c4c0fc346c9c5bc2d74